src/app/[locale]/list/_components/AasList.tsx

```diff
@@ -9,25 +9,38 @@
     Typography,
     useTheme,
 } from '@mui/material';
-import { FormattedMessage } from 'react-intl';
+import { FormattedMessage, useIntl } from 'react-intl';
 import { messages } from 'lib/i18n/localization';
 import CompareArrowsIcon from '@mui/icons-material/CompareArrows';
-import { AasListEntry } from 'lib/api/generated-api/clients.g';
 import { AasListTableRow } from 'app/[locale]/list/_components/AasListTableRow';
+import { AasListDto } from 'lib/services/list-service/ListService';
 
 type AasListProps = {
-    shells: AasListEntry[];
-    tableHeaders: { label: string }[];
+    shells: AasListDto | undefined;
     comparisonFeatureFlag?: boolean;
     selectedAasList: string[] | undefined;
     updateSelectedAasList: (isChecked: boolean, aasId: string | undefined) => void;
 };
 
 export default function AasList(props: AasListProps) {
```

Comment:
you can inline the arguments
```suggestion
export default function AasList({ shells, selectedAasList, updateSelectedAasList, comparisonFeatureFlag }: AasListProps) {
```